flutter/lib/src/navigation/sentry_navigator_observer.dart

```diff
@@ -100,6 +115,32 @@ class SentryNavigatorObserver extends RouteObserver<PageRoute<dynamic>> {
       });
     }
   }
+
+  void _startTransaction(String? name, Object? arguments) {
+    if (!_enableAutoTransactions) {
+      return;
+    }
+    if (name == null) {
+      return;
+    }
+    _transaction = _hub.startTransaction(
+      name,
+      'ui.load',
+      bindToScope: true,
```

Comment:
same on Android, we don't pass `bindToScope` since default is false, and call `configureScope` manually, and check if `scope.transaction` is null before setting it

Reply:
Good points, will change!

Reply:
We are now checking `configureScope` and only starting the transaction if there is no other.

Reply:
the transaction is created regardless, only setting to the scope or not that changes, See https://github.com/getsentry/sentry-java/blob/main/sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java#L192-L209

Reply:
I evaluate this using `scope.span`, and therefore only if a "real" transaction is present, not just a string, as we also set the current route as transaction name shortly before starting a transaction.